book/getting-started/linux-setup.md

```diff
@@ -57,15 +57,21 @@ sudo apt-get install -y build-essential
 sudo apt-get install -y libssl-dev
 ```
 
-### Install GCC 5 or Higher
+### Install GCC 4.7 or Higher
 
- If you have at least GCC 5 installed on your machine, you don't need to do anything. If you have gcc 4.9 or lower, you'll need to upgrade. You can check your `gcc` version by running:
+ If you have at least GCC 4.7 installed on your machine, you don't need to do anything. If you have gcc 4.6 or lower, you'll need to upgrade. You can check your `gcc` version by running:
 
 ```bash
 gcc --version
 ```
 
-To upgrade to GCC 5:
+To upgrade to GCC 5 if you don't have at least GCC 4.7:
+
+```bash
+sudo apt-get install software-properties-common
```

Comment:
Shouldn't this also be added to `book/go/getting-started/linux-setup.md`?

Reply:
Good catch @dipinhora, coming up.